samples/sensor/bme68x/prj.conf

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_SENSOR=y
+CONFIG_SENSOR_INFO=y
+
+CONFIG_BME680=n
```

Comment:
Why is this necessary? Is it enabled because of the device tree entry?

Reply:
It's enabled by default in it's KConfig definition.